presets/4.5/tune/karate/karate_whoop_late_2024.txt

```diff
@@ -0,0 +1,183 @@
+#$ TITLE: Karate whoop late 2024
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: TUNE
+#$ STATUS:  EXPERIMENTAL
+#$ KEYWORDS: karate, 1S, race, brushless, whoop, 65mm, sugarK, KarateBrot
+#$ AUTHOR: sugarK
+
+#$ PARSER: MARKED
+
+#$ DESCRIPTION:
+#$ DESCRIPTION: <img src="https://user-images.githubusercontent.com/19867640/174759844-ffd85f42-1f1c-42bf-9032-d9f96d4d9619.svg" width="90px" style="display: block; float: left; margin-right: 10px; margin-top: 20px"/>
+#$ DESCRIPTION: <h1>Karate Race 6S 5"</h1>
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION: This 1s whoop racing tune was developed using my custom Betafpv 5n1 build that weighs 15.6g [Air Brushless FC](https://betafpv.com/collections/brushless-flight-controller/products/air-brushless-flight-controller?variant=41142912745606) it is a slider tune allowing easy modification. 
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION: This tune should work well on any BRUSHLESS powered quality whoop build in the sub 20g range. It shouldn't be used on a brushed whoop.
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION: ## Things to note:
+#$ DESCRIPTION: - **YOU HAVE TO USE RPM FILTERING WITH THIS TUNE!** Failure to do so might result in fire 🔥
+#$ DESCRIPTION: - Also this should be applied to a clean flash after you've setup your rates, swtiches, vtx tables etc.
+#$ DESCRIPTION: - To test arm the quad with props on, it should **sound clean with no grinding**. If it passes that then hover test and check motor temps.
+#$ DESCRIPTION: - **If anything is off, don't fly it!!**
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION: ## Second note... Radio links:
+#$ DESCRIPTION: 1. Make sure your radio firmware is up to date using either EdgeTX or OpenTX
+#$ DESCRIPTION: 1. Make sure your **ADC Filter is OFF** in the hardware page
+#$ DESCRIPTION: 1. Go to the radio (RC_LINK) presets and apply the correct setup for your system and link speed
+
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/490
+
+#$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
+#$ INCLUDE: presets/4.4/tune/defaults.txt
+
+# --DSHOT --
+set motor_pwm_protocol = Dshot300
+
+# -- Gyro lowpass filters --
+set gyro_lpf1_dyn_min_hz = 0
+set gyro_lpf1_dyn_max_hz = 0
+set gyro_lpf1_static_hz = 0
+
+# -- Gyro Dynamic Notches --
+set dyn_notch_count = 1
+set dyn_notch_q = 600
+set dyn_notch_max_hz = 750
+
+# -- Dterm filtering --
+set dterm_lpf1_dyn_expo = 7
+
+# -- RPM filtering --
+set motor_poles = 12
+set dshot_bidir = ON
+set rpm_filter_fade_range_hz = 150
+set rpm_filter_harmonics = 1
+
+# -- iTerm  --
+set iterm_relax_cutoff = 45
+
+# -- Misc --
+set thrust_linear = 20
+set vbat_max_cell_voltage = 445
+
+#$ OPTION BEGIN (CHECKED): Dynamic idle for sub 20g whoop ( recommended )
+    set dyn_idle_min_rpm = 150
+    set dyn_idle_p_gain = 35
+    set dyn_idle_start_increase = 60
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Vbat sag compensation ( recommended )
```

Comment:
intended to be checked? I don't remember if sag compensation is for throttle, or PIDs adjsutments?

Reply:
pids and yeah you should use it